resources/views/auth/adminlogin.blade.php

```diff
@@ -0,0 +1,106 @@
+{{-- Login Register CSS --}}
+<link rel="stylesheet" href="{{ asset('css/authpage.css') }}">
+
+<script src="/js/app.js"></script>
+
+@extends('layouts.app')
+
+@section('title', 'Sign in')
+
+@section('content')
+<div class="container-fluid" style="background-image: url(/images/account/bg-admin.png)">
+    <!-- Logo-->
+    <div class="row">
+        <div class="col-md-6 mb-3">
+            <div class="">
+                <div class="ms-0">
+                    <a  href="{{ url('/') }}" class="logo-img">
+                        <img src="images/common/Logo.png" alt="logo" style="width: 150px; height:150px;">
```

Comment:
Please use asset when access images like this below:

https://github.com/kredo-school/18tha-gj_mall/pull/14/files#diff-12e05a6ae0b6a6da563eb899687b9da0d94073554ad4041f40d6cbb2816688e5R32